src/main/java/com/google/gfx/services/matching/MatchingService.java

```diff
@@ -0,0 +1,70 @@
+package com.google.gfx.services.matching;
+
+import com.google.gfx.model.Match;
+import com.google.gfx.model.Order;
+import com.google.gfx.model.User;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+/**
+ * Matches existing orders with other existing or new ones.
+ */
+public interface MatchingService {
+
+  /**
+   * This method was conceived to be ran as part of a cron job that matches existing buy and sell
+   * orders.
+   */
+  void matchExistingBuyAndSellOrders();
```

Comment:
I'm still a bit divided about whether to keep this, but will do for now so this doesn't fall out of the radar.